flytekit/bin/entrypoint.py

```diff
@@ -122,38 +122,15 @@ def execute_task_cmd(task_module, task_name, inputs, output_prefix, raw_output_d
 
 
 @_pass_through.command("pyflyte-fast-execute")
-@_task_module_option
-@_task_name_option
-@_inputs_option
-@_output_prefix_option
-@_raw_output_date_prefix_option
-@_test
 @_click.option("--additional-distribution", required=False)
-@_click.option("--virtual-env", multiple=True, help="Virtual envs to invoke the command with")
-def fast_execute_task_cmd(
-    task_module, task_name, inputs, output_prefix, raw_output_data_prefix, test, additional_distribution, virtual_env
-):
+@_click.argument("task-execute-cmd", nargs=-1, type=_click.UNPROCESSED)
```

Comment:
can you write an example of how this will be passed in - I think it will be
`fast-execute -- pyflyte execute ....`

Reply:
this is meant to work with the fast_register call, so that method fills it in using whatever underlying container args originally exist